puppet/lib/puppet/util/puppetdb/command.rb

```diff
@@ -42,7 +42,7 @@ def initialize(command, version, certname, payload)
       #
       # This is roughly inline with how Puppet serializes for catalogs as of
       # Puppet 4.1.0. We need a better answer to non-utf8 data end-to-end.
-      }.to_pson)
+      }.to_pson, "Error encoding a '#{command}' command for host '#{certname}'")
```

Comment:
should I see this getting logged with this catalog?
```
exec {"\u006E\u0303":
  command => "/usr/bin/uptime"
}

exec {"\u00F1":
  command => "/usr/bin/date"
}
```

I see this:
```
Warning: Ignoring invalid UTF-8 byte sequences in data to be sent to PuppetDB
```
in the puppet log, but no mention of replace catalog specifically.